utils/resources/resources.go

```diff
@@ -0,0 +1,178 @@
+package resources
+
+import (
+	"context"
+	"github.com/kiegroup/container-builder/client"
+	v08 "github.com/kiegroup/kogito-serverless-operator/api/v1alpha08"
+	"github.com/kiegroup/kogito-serverless-operator/utils/kubernetes"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"net/http"
+	"os"
+	"path/filepath"
+	ctrl "sigs.k8s.io/controller-runtime/pkg/client"
+	"strings"
+)
+
+// ResourceCustomizer can be used to inject code that changes the objects before they are created.
+type ResourceCustomizer func(object ctrl.Object) ctrl.Object
+
+// IdentityResourceCustomizer is a ResourceCustomizer that does nothing.
```

Comment:
what's the purpose of it?

Reply:
This is a way to pass a function as argument of another function (used in installBuilderServiceAccountRolesKubernetes or installBuilderServiceAccountRolesKubernetes)

Reply:
I've asked because of this description, "does noting"